docs/codeql/codeql-cli/about-codeql-packs.rst

```diff
@@ -56,49 +59,282 @@ The following properties are supported in ``qlpack.yml`` files.
 
    * - Property
      - Example
-     - Required
+     - Required by
      - Purpose
    * - ``name``
-     - ``octo-org/security-queries``
+     - .. code-block:: yaml
```

Comment:
I'm not sure including code blocks tables this way will work. The Docs/Sphinx CI (if it runs) check should pick it up though.

Reply:
I checked locally and it is working, but I will be sure to check again.

Reply:
Ah, I've also checked the preview and you're right. Ignore me.

Reply:
I'll resolve the other comments related to this.